packages/opencensus-core/src/stats/view.ts

```diff
@@ -200,9 +200,10 @@ export class BaseView implements View {
 
   /**
    * Gets view`s metric
+   * @param start The start timestamp
```

Comment:
Should this say add "in epoch milliseconds"?

Reply:
Done,thanks